### PR TITLE
69 translate get statistics

### PR DESCRIPTION
**Pull request by cbocionek**
Relates to issue #69. 
Adds possibility of translating short and long statistic descriptions in result of `get_statistics` into different languages by using optional argument `target_language`, e.g. `get_statistics(target_language='en')`.
Added static file for translation of short and long statistic descriptions into English. Translation into other languages is currently not statically available. Optionally, an instance of a class implementing  `TranslationProvider` can be passed to `get_statistics` to translate its result on-the-fly.
By default, an instance of `SchemaTranslationProvider` is used, which accesses the static translation file.

The script `tasks/translate_schema.py` was used to generate the static translation file and can be reexecuted as needed. Uses the package `googletrans`, which was added as a development dependency.

### CODE REVIEW

File: tasks/translate_schema.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    for source_texts in statistic_descriptions.values():
+        for source_text in source_texts:
+            if source_text not in translation_dictionary:
+                translated_text = translation_provider.translate_from_german(source_text, target_language)
+                translation_dictionary[source_text] = translated_text
+                time.sleep(sleep_duration_between_requests_in_seconds)
+
+    return {target_language: translation_dictionary}
+
+
```
Comment (enryH) [inline tasks/translate_schema.py:62]: not sure if the following should be factored into an example?

File: tasks/translate_schema.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    metadata_provider = StatisticsSchemaJsonMetaDataProvider()
+    statistic_descriptions = metadata_provider.get_stat_descriptions()
+    translation_dictionary = dict()
+
+    for source_texts in statistic_descriptions.values():
+        for source_text in source_texts:
+            if source_text not in translation_dictionary:
+                translated_text = translation_provider.translate_from_german(source_text, target_language)
+                translation_dictionary[source_text] = translated_text
+                time.sleep(sleep_duration_between_requests_in_seconds)
```
Comment (enryH) [inline tasks/translate_schema.py:58]: Maybe we can check how long some of the example queries will take to be translated if 2 seconds are to be waited?

Comment (cbocionek) [reply tasks/translate_schema.py:58]: The execution time of the requests themselves is negligible. The wait between requests aims at reducing the probability of getting blocked for sending too many requests too fast when creating the translation dictionary.

Comment (enryH) [reply tasks/translate_schema.py:58]: How many requests are needed for a simple query? Is there a restriction on the length of the translation query (then we could maybe group sentences to translate first into paragraphs?) I am just brainstorming as I have still to checkout  what is in a `translation_dictionary`:)

**Comment by KonradUdoHannes** [comment]
While reviewing the pull request I noticed an issue regarding the pypi deployment with respect to the package data. This turned out to be a critical bug in the 0.2.1 release. I fixed this and released 0.2.2 such that the current release is working again. It meant some slight change in the package structure that is also essential for the current pull request to work as a pypi release. As such I would recommend @cbocionek  to merge the current master into the branch and adjust file paths accordingly.

For the future I hope that we'll find some additional configuration/tool for qa that will help to catch this type of pypi deployment problem automatically and hence earlier.

**Pull request merged**